From 680ba0035d25eda89047f6945e4cf2426f7d0afe Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Robert-Andr=C3=A9=20Mauchin?= <zebob.m@gmail.com>
Date: Tue, 9 Jul 2019 22:10:37 +0200
Subject: [PATCH] Revert 372b3438e5f995f14d9317352b432dc43cfdd40d

---
 cmd/promtool/main.go | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/cmd/promtool/main.go b/cmd/promtool/main.go
index ffae6991..9ff7bf92 100644
--- a/cmd/promtool/main.go
+++ b/cmd/promtool/main.go
@@ -350,7 +350,7 @@ func QueryInstant(url, query string, p printer) int {
 	api := v1.NewAPI(c)
 
 	ctx, cancel := context.WithTimeout(context.Background(), 2*time.Minute)
-	val, _, err := api.Query(ctx, query, time.Now()) // Ignoring warnings for now.
+	val, err := api.Query(ctx, query, time.Now())
 	cancel()
 	if err != nil {
 		fmt.Fprintln(os.Stderr, "query error:", err)
@@ -419,7 +419,7 @@ func QueryRange(url string, headers map[string]string, query, start, end string,
 	api := v1.NewAPI(c)
 	r := v1.Range{Start: stime, End: etime, Step: step}
 	ctx, cancel := context.WithTimeout(context.Background(), 2*time.Minute)
-	val, _, err := api.QueryRange(ctx, query, r) // Ignoring warnings for now.
+	val, err := api.QueryRange(ctx, query, r)
 	cancel()
 
 	if err != nil {
@@ -473,7 +473,7 @@ func QuerySeries(url *url.URL, matchers []string, start, end string, p printer)
 	// Run query against client.
 	api := v1.NewAPI(c)
 	ctx, cancel := context.WithTimeout(context.Background(), 2*time.Minute)
-	val, _, err := api.Series(ctx, matchers, stime, etime) // Ignoring warnings for now.
+	val, err := api.Series(ctx, matchers, stime, etime)
 	cancel()
 
 	if err != nil {
-- 
2.21.0

